projects/kit/components/input-month/input-month.directive.ts

```diff
@@ -0,0 +1,136 @@
+import {
+    computed,
+    Directive,
+    effect,
+    inject,
+    INJECTOR,
+    Input,
+    signal,
+} from '@angular/core';
+import {toSignal} from '@angular/core/rxjs-interop';
+import {tuiAsControl, TuiControl} from '@taiga-ui/cdk/classes';
+import {TUI_ALLOW_SIGNAL_WRITES} from '@taiga-ui/cdk/constants';
+import type {TuiMonth} from '@taiga-ui/cdk/date-time';
+import {TUI_FIRST_DAY, TUI_LAST_DAY} from '@taiga-ui/cdk/date-time';
+import {tuiFallbackValueProvider} from '@taiga-ui/cdk/tokens';
+import {tuiValueBinding} from '@taiga-ui/cdk/utils/dom';
+import {tuiDirectiveBinding} from '@taiga-ui/cdk/utils/miscellaneous';
+import {
+    TUI_TEXTFIELD_OPTIONS,
+    tuiInjectAuxiliary,
+    TuiWithTextfield,
+} from '@taiga-ui/core/components/textfield';
+import {TUI_DROPDOWN_OPTIONS, tuiDropdownOpen} from '@taiga-ui/core/directives/dropdown';
+import {TuiIcons} from '@taiga-ui/core/directives/icons';
+import type {TuiCalendarMonth} from '@taiga-ui/kit/components/calendar-month';
+import {TUI_MONTH_FORMATTER} from '@taiga-ui/kit/tokens';
+
+import {TUI_INPUT_MONTH_OPTIONS} from './input-month.options';
+import {TuiNativeMonthPicker} from './native-month-picker/native-month-picker.component';
+
+@Directive({
+    standalone: true,
+    selector: 'input[tuiInputMonth]',
+    providers: [tuiAsControl(TuiInputMonthDirective), tuiFallbackValueProvider(null)],
+    hostDirectives: [TuiWithTextfield],
+    host: {
+        '(click)': 'toggleDropdown()',
+        '(blur)': 'onTouched()',
+        '[disabled]': 'disabled()',
+        // Make <input /> readonly (without readonly appearance)
```

Comment:
Not sure we need this comment here, how about we add [inputmode='none'] { caret-color: transparent } into textfield styles? Do we ever need it with visible caret?

Reply:
https://github.com/taiga-family/taiga-ui/pull/10345/commits/c2580fce6cc9ac7226bc714eeca3eaae1dc34286